src/sentry/issues/grouptype.py

```diff
@@ -43,6 +68,51 @@ def __post_init__(self) -> None:
         if self.category not in valid_categories:
             raise ValueError(f"Category must be one of {valid_categories} from GroupCategory.")
 
+        # ensure that noise config ignore limits only increase with rollouts
+        # use a ratio of ignore limit to expiry time to allow for greater flexibility
+        if self.group_policy:
+            prev_ignore_limit_ratio = 0
+            group_policy = self.group_policy
+            if group_policy.limited_access:
+                prev_ignore_limit_ratio = (
+                    group_policy.limited_access.ignore_limit
+                    / group_policy.limited_access.expiry_time
+                )
+
+            if group_policy.early_access:
+                early_access_ratio = (
+                    group_policy.early_access.ignore_limit / group_policy.limited_access.expiry_time
+                )
+                if early_access_ratio < prev_ignore_limit_ratio:
+                    raise ValueError(
+                        "Early Access ignore limit ratio must be greater than Limited Access ignore limit ratio"
+                    )
+                prev_ignore_limit_ratio = early_access_ratio
+
+            general_access_ratio = (
+                group_policy.general_access.ignore_limit / group_policy.general_access.expiry_time
+            )
+            if general_access_ratio < prev_ignore_limit_ratio:
+                raise ValueError(
+                    "General Access ignore limit must be greater than Early Access and Limited Access ignore limit ratios"
+                )
+
+
+def get_noise_config(
+    grouptype: Type[GroupType], organization: Organization
+) -> Optional[NoiseConfig]:
+    group_policy = grouptype.group_policy
+    if not group_policy:
+        return None
```

Comment:
need to think through this a bit more...I like the idea of needing to specify group policy if you want to use noise reduction, but not sure that it will transition well when a detector moves out of limited access to general access

Reply:
So right now if you put `group_policy=GroupPolicy()` that's all that's needed to enable noise reduction, otherwise the default is off